src/Resources/assets/controllers/accordion_controller.js

```diff
@@ -63,5 +90,19 @@ export default class extends Controller {
 
         return element.parentElement;
     }
+
+
+    /**
+     * @param {HTMLElement|null} element
+     * @param {string} target
+     */
+    closestChildTarget(element, target) {
+        const childElement = element.querySelector(`[data-araise--table-bundle--accordion-target='${target}']`);
+        if(childElement) {
+            return childElement;
+        }
+
+        return null;
```

Comment:
Nit: In case this is is an expected output, that's fine. Otherwise, there should either be an error thrown (`throw new Error(...)` for the case, this has to be fixed) or an error provided via `console.error(...)` (in case it isn't a severe problem for usability).